duneggd/ArgonCube/NDBucket.py

```diff
@@ -169,5 +169,4 @@ def construct(self,geom):
                                                             pos=FieldcageGap_pos,
                                                             copynumber=2*ii*jj)
 
-                ArgonColumn_lv.placements.append(FieldcageGap_pla.name)
-
+                ArgonColumn_lv.placements.append(FieldcageGap_pla.name)
```

Comment:
Your editor seems to not add newlines at the end of files. Could you fix this as well?